src/ethereum_test_rpc/types.py

```diff
@@ -136,9 +136,22 @@ def blob_versioned_hashes(self) -> List[Hash]:
         return [Hash(b"\1" + commitment[1:]) for commitment in self.commitments]
 
 
+class BlobAndProof(CamelModel):
+    """Represents a blob and proof structure."""
+
+    blob: Bytes
+    proofs: List[Bytes]
```

Comment:
We might need to implement `BlobAndProofV1` too to verify Cancun/Prague from this: https://github.com/ethereum/execution-apis/blob/main/src/engine/cancun.md#blobandproofv1

So we can have:
```suggestion
class BlobAndProof(CamelModel):
    """Represents a blob and proof structure."""

    blob: Bytes
    proofs: List[Bytes] | None = None
    proof: Bytes | None= None
```
Or as an alternative we could simply define `class BlobAndProofV1` and `class BlobAndProofV2` each with the expected fields.